cypress/e2e/court-cases/caseDetails.cy.ts

```diff
@@ -49,10 +50,10 @@ describe("Case details", () => {
 
       cy.injectAxe()
 
-      cy.checkA11y(undefined, a11yConfig, logAccessibilityViolations)
+      cy.checkA11y("#content", a11yConfig, logAccessibilityViolations)
```

Comment:
This test was continually failing as it was returning empty html when defined when it should grab the whole page, this gets the <Main /> element and asserts on that explicitly.

Reply:
I think we want to run accessibility tests on the entire page in e2e tests, not just this element. Bear in mind accessibility tests are failing in dev mode because NEXT doesn't render some of the landmarks correctly. You can try npm run build && npm run start which should pass these tests